WordPress/src/main/java/org/wordpress/android/ui/main/MeFragment.kt

```diff
@@ -85,9 +77,7 @@ class MeFragment : Fragment(R.layout.me_fragment), OnScrollToTopListener {
     @Inject lateinit var meGravatarLoader: MeGravatarLoader
     @Inject lateinit var viewModelFactory: ViewModelProvider.Factory
     @Inject lateinit var mediaPickerLauncher: MediaPickerLauncher
-    @Inject lateinit var recommendTheAppFeatureConfig: RecommendTheAppFeatureConfig
     @Inject lateinit var sequencer: SnackbarSequencer
-    @Inject lateinit var unifiedAboutFeatureConfig: UnifiedAboutFeatureConfig
```

Comment:
Since these are not being used anywhere anymore we can probably go ahead and remove them as well? I'd just check with the folks that worked on the "recommend the app" feature to double-check this is ok.

Reply:
Thank you @renanferrari , Removed them as well and pinged about Recommend The App.  Also moved this to 19.1